tarpc/src/macros.rs

```diff
@@ -569,6 +581,20 @@ mod functional_test {
         assert_eq!(3, client2.add(1, 2).get().unwrap());
     }
 
+    #[test]
+    fn async_try_clone_unix() {
+        let handle = Server.spawn_with_config(UnixTransport("/tmp/test"),
```

Comment:
I don't think tempdir exposes the path? I'll double check.
On Feb 24, 2016 7:40 PM, "shaladdle" <notifications@github.com> wrote:

> In tarpc/src/macros.rs
> <https://github.com/google/tarpc/pull/27#discussion_r54045195>:
>
> > @@ -569,6 +581,20 @@ mod functional_test {
> >          assert_eq!(3, client2.add(1, 2).get().unwrap());
> >      }
> >
> > +    #[test]
> > +    fn async_try_clone_unix() {
> > +        let handle = Server.spawn_with_config(UnixTransport("/tmp/test"),
>
> Use this: https://crates.io/crates/tempdir? I think that generates random
> directories, so you can do Tempdir::new().unwrap().path().join("file");
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/tarpc/pull/27/files#r54045195>.
>